server/lib/api/transaction.js

```diff
@@ -132,7 +133,54 @@ module.exports = function transactionAPI(router) {
           }
         });
     } else if (req.query.address) {
+      getBlock(
+        { $or:
+          [
+            { 'txs.outputs.address':     req.query.address },
+            { 'txs.inputs.prevout.hash': req.query.address },
+          ],
+        },
+        { rawBlock: 0 },
+        MAX_BLOCKS,
+        (err, block) => {
+          if (err) {
+            res.status(501).send();
+            logger.log('err', err);
+          }
 
+          if (block[0]) {
```

Comment:
return early - 
```
if (!block[0]){
  return res.send([]);
}
```
etc.. then no additional conditionals needed and less nesting